packages/dev-utils/src/chai_revert_error.ts

```diff
@@ -0,0 +1,198 @@
+import { AnyRevertError, RevertError, StringRevertError } from '@0x/utils';
+
+// tslint:disable only-arrow-functions prefer-conditional-expression
+
+type ChaiPromiseHandler = (x: any, ...rest: any[]) => Promise<void>;
+type ChaiAssertHandler = (x: any, ...rest: any[]) => void;
+
+interface Chai {
+    Assertion: ChaiAssertionType;
+}
+
+interface ChaiAssertionInstance {
+    assert: ChaiAssert;
+    _obj: any;
+    __flags: any;
+}
+
+interface ChaiAssertionType {
+    overwriteMethod: (name: string, _super: (expected: any) => any) => void;
+    new (): ChaiAssertionInstance;
+}
+
+type ChaiAssert = (
+    condition: boolean,
+    failMessage?: string,
+    negatedFailMessage?: string,
+    expected?: any,
+    actual?: any,
+) => void;
+
+interface GanacheTransactionRevertResult {
+    error: 'revert';
+    program_counter: number;
```

Comment:
Our convention is to always use camel-case. Is this needed to satisfy a chain interface?

Reply:
Yup. Those are the properties on a Ganache revert, so it's out of our control.